### PR TITLE
Add debianization for Bullseye [REVPI-3011]

### PR DESCRIPTION
**Pull request by nbuchwitz**

### CODE REVIEW

**Comment by nbuchwitz** [comment]
I've added the merge for #10 to this PR

**Review by iluminat23 — CHANGES_REQUESTED** [review]
The changelog changes should be done after the merge. Then the issue about wrong entries in the changelog is avoided.
You can also just do the whole Add debianization for bullseye after the merge. Also add debianization for bullseye is not correct Update debianzation for Bullseye would be correct.

File: debian/changelog
Comment (iluminat23) [inline debian/changelog]: https://github.com/RevolutionPi/revpi-repo/commit/ef61cf6d5c90c7e30d6c82d8bf7f24fdf208ae5a
gbp dch should prevent this ;)
```suggestion
  * Update package list to Bullseye
```

**Review by iluminat23 — CHANGES_REQUESTED** [review]

**Review by naruxde — APPROVED** [review]
<3

**Review by iluminat23 — APPROVED** [review]

**Comment by iluminat23** [comment]
Keep in mind, don't merge with git hub UI ;)

**Pull request merged**
